src/serac/infrastructure/git_sha.hpp.in

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2019-2022, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#ifndef SERAC_GIT_SHA_HPP
+#define SERAC_GIT_SHA_HPP
```

Comment:
we've been using mostly `#pragma once` in other header files, is there a reason for the `#ifndef ... ` + `#define ...` pattern here?

Reply:
copy paste... good catch ill update it